### PR TITLE
[BE] Feature/patro23 53 request/response logging middleware

### PR DESCRIPTION
**Pull request by Kepeczsz**
https://tracker.intive.com/jira/secure/RapidBoard.jspa?rapidView=1665&view=detail&selectedIssue=PATRO23-53

### CODE REVIEW

File: src/api/app/appsettings.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
-    }
+		"LogLevel": {
```
Comment (mateuszmuszynski) [inline src/api/app/appsettings.json:3]: Incorrect indentation

File: src/api/app/Program.cs [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+	logging.RequestBodyLogLimit = 4096;
+	logging.ResponseBodyLogLimit = 4096;
+});
 
 var app = builder.Build();
 
 app.UseExampleModule();
-
+app.UseHttpLogging();
 app.Run();
```
Comment (mateuszmuszynski) [inline src/api/app/Program.cs:18]: Add new line at the end of file

**Review by mateuszmuszynski — CHANGES_REQUESTED** [review]

File: src/api/app/appsettings.json (hunk 1 above)
Comment (Kepeczsz) [reply src/api/app/appsettings.json:3]: i do not understand, how that's incorrect indentation, can u provide me more information?
It seems like github has in his view indentation of 4, whereas i use standard option for js formatting of 2.
and it looks like it on my pc.
obraz_2023-03-12_222042195

Comment (Kepeczsz) [reply src/api/app/appsettings.json:3]: @mateuszmuszynski

Comment (mateuszmuszynski) [reply src/api/app/appsettings.json:3]: It looks like there are spaces and tabs in this file. Please use spaces OR tabs.
image

Comment (Kepeczsz) [reply src/api/app/appsettings.json:3]: @mateuszmuszynski  i've refactored file to use only tabs, also merged dev into my branch, to resolve merge conflicts.

File: src/api/app/Program.cs (hunk 2 above)
Comment (mateuszgiedzinski) [reply src/api/app/Program.cs]: @mateuszmuszynski  relates to our stylecop rule ends file with new line is incorrect. But I suggest to add new line between registration like below.
app.UseExampleModule

app.UseHttpLogging

app.Run

File: src/api/app/Program.cs [hunk 3]
```diff
@@ -34,5 +41,5 @@
 }
 
 app.UseExampleModule();
-
+app.UseHttpLogging();
```
Comment (mateuszgiedzinski) [inline src/api/app/Program.cs:44]: I suggest separate UseHttpLogging() UseModule..() and Run() by empty line.

**Review by mateuszgiedzinski — CHANGES_REQUESTED** [review]

**Review by mateuszgiedzinski — APPROVED** [review]

**Review by mateuszmuszynski — APPROVED** [review]

**Pull request merged**